README.md

```diff
@@ -4,71 +4,76 @@ This package is used to create a connection to Cognigy.AI via a Socket Endpoint.
-| typingStatus | `"on"` or `"off"` | fires when the typing indicator should show or hide
-| finalPing | - | fires when the bot is done processing a message
-| error | `{ message }` | fires when an error happened in the bot
+
+| Name         | Event Payload     | Description                                         |
+| ------------ | ----------------- | --------------------------------------------------- |
+| output       | `{ text, data }`  | fires on every incoming message from the bot        |
+| typingStatus | `"on"` or `"off"` | fires when the typing indicator should show or hide |
+| finalPing    | -                 | fires when the bot is done processing a message     |
+| error        | `{ message }`     | fires when an error happened in the bot             |
 
 ## Options
+
 You can pass a third argument to `SocketClient` to set additional options as follows:
 
 ```javascript
-const client = new SocketClient('https://socket.url', 'socket-token', {
-    userId: 'user1234'
+const client = new SocketClient("https://socket.url", "socket-token", {
+  userId: "user1234",
 });
 ```
 
-| Name | Type | Default | Description '
-| - | - | - | - |
-| `userId` | string | random string | the user id for the conversation
-| `sessionId` | string | random string | the session id for the conversation
-| `channel` | string | `"socket-client"` | the name of the channel (can be used for analytics purposes)
-| `forceWebsockets` | boolean | `false` | if this is enabled, there will be no fallback to http polling
-| `interval` | number | `10000` | the interval for polling if in http polling fallback
-| `reconnection` | boolean | `true` | if enabled, will try to reconnect if the connection is aborted
-| `reconnectionLimit` | number | `5` | limit the maximum number of reconnection attempts, `0` means no limit
-
+| Name                | Type    | Default                                | Description '                                                                            |
+| ------------------- | ------- | -------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `userId`            | string  | random string                          | the user id for the conversation                                                         |
+| `sessionId`         | string  | random string                          | the session id for the conversation                                                      |
+| `channel`           | string  | `"socket-client"`                      | the name of the channel (can be used for analytics purposes)                             |
+| `forceWebsockets`   | boolean | auto-determined by runtime-environment | if this is enabled, there will be no fallback to http polling (wins over `forcePolling`) |
+| `forcePolling`      | boolean | `false`                                | if this is enabled, there will be no upgrade to websockets                               |
```

Comment:
I feel like this can be formulated more clearly, like "websockets will not be used".

Reply:
I'm referring to the "there will be no upgrade to websockets" part

Reply:
fixed